docs/pages/base/api/use-input.json

```diff
@@ -33,7 +33,59 @@
     },
     "value": { "type": { "name": "unknown", "description": "" } }
   },
-  "returnValue": {},
+  "returnValue": {
+    "disabled": {
+      "type": { "name": "boolean", "description": "If `true`, the component will be disabled." },
+      "default": "false",
+      "required": true
+    },
+    "error": {
+      "type": {
+        "name": "boolean",
+        "description": "If `true`, the `input` will indicate an error by setting the `aria-invalid` attribute."
+      },
+      "default": "false",
+      "required": true
+    },
+    "focused": {
+      "type": { "name": "boolean", "description": "If `true`, the `input` will be focused." },
+      "default": "false",
+      "required": true
+    },
+    "formControlContext": {
+      "type": {
+        "name": "FormControlUnstyledState | undefined",
+        "description": "Return value from the `useFormControlUnstyledContext` hook."
+      },
+      "required": true
+    },
+    "getInputProps": {
+      "type": {
+        "name": "<TOther extends Record<string, any> = {}>(externalProps?: TOther) => UseInputInputSlotProps<TOther>",
+        "description": "Resolver for the input slot's props."
+      },
+      "required": true
+    },
+    "getRootProps": {
+      "type": {
+        "name": "<TOther extends Record<string, any> = {}>(externalProps?: TOther) => UseInputRootSlotProps<TOther>",
+        "description": "Resolver for the root slot's props."
```

Comment:
```suggestion
        "description": "Resolver for the `input` slot's props."
      },
      "required": true
    },
    "getRootProps": {
      "type": {
        "name": "<TOther extends Record<string, any> = {}>(externalProps?: TOther) => UseInputRootSlotProps<TOther>",
        "description": "Resolver for the `root` slot's props."
```

@samuelsycamore one last question :) Should we consider the slots' names as code elements too?

Reply:
Good question! I have not been styling slot names in any special way—here's an example from the Base docs:

<img width="862" alt="Screenshot 2023-02-06 at 10 01 12 AM" src="https://user-images.githubusercontent.com/71297412/217021549-d8790a30-341c-43d6-8cd9-909d67e74b2d.png">

Reply:
I would say it makes sense to use back ticks when referring specifically to the class name: so either "the root slot" or `MuiSwitch-root`.

Reply:
Alright, in this case I am merging the PR, no need for additional changes.